packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1014,6 +1065,27 @@ class RawScrollbar extends StatefulWidget {
   /// {@endtemplate}
   final bool? isAlwaysShown;
 
+  /// The [shape] of the scrollbar's thumb.
+  ///
+  /// If you need just a simple rectangle with rounded borders, use [radius].
+  /// Using [shape] and [radius] together will not work and will throw an
+  /// error. Use only one (or none) of them to construct the [RawScrollbar].
+  ///
+  /// If neither [shape] nor [radius] are specified, the thumb will take the
+  /// shape of a regular rectangle whose width is defined by [thickness], which
+  /// defaults to 6.0. Otherwise, the thumb will take the shape of the
+  /// [OutlinedBorder] and fill itself with [thumbColor] (or grey if it
+  /// is unspecified).
+  ///
+  /// The [shape] could be any one of the subclasses of [OutlinedBorder].
```

Comment:
I think it would be more helpful to explain how to inset a shape within a RawScrollbar that has a border. An example would be useful.